front/pages/w/[wId]/assistant/assistants.tsx

```diff
@@ -183,201 +108,94 @@ export default function PersonalAssistants({
           mutateAgentConfigurations={mutateAgentConfigurations}
         />
       )}
-
-      {showRemovalModal && (
-        <RemoveAssistantFromListDialog
-          owner={owner}
-          agentConfiguration={showRemovalModal}
-          show={!!showRemovalModal}
-          onClose={() => setShowRemovalModal(null)}
-          onRemove={() => {
-            void mutateAgentConfigurations();
+      <div className="flex flex-col gap-4 pt-9">
+        <Searchbar
+          name="search"
+          size="md"
+          placeholder="Assistant Name"
```

Comment:
Should we be consistent with the gallery screen? If yes, should be "Search (Name)".